examples/table-queries-join/table_queries_join.bal

```diff
@@ -0,0 +1,95 @@
+import ballerina/io;
+
+// This `record` type represents a person.
+type Person record {
+    int id;
+    int age = -1;
+    float salary;
+    string name;
+    boolean married;
+};
+
+// This `record` type represents an order.
+type Order record {
+    int personId;
+    int orderId;
+    string items;
+    float amount;
+};
+
+// This `record` type represents the order details (this is derived by joining the person details
+//and the order details).
+type OrderDetails record {
+    int orderId;
+    string personName;
+    string items;
+    float amount;
+};
+
+// This is the main function.
```

Comment:
Shall we remove this? Some time back we decided to leave out comments like this that doesn't add something more than what the user can deduce by looking at the code.